lib/ruby_smb/dcerpc/alter_context.rb

```diff
@@ -0,0 +1,28 @@
+module RubySMB
+  module Dcerpc
+    class AlterContext < BinData::Record
```

Comment:
Just adding a reference to the documentation:
```suggestion
    # The Alter context PDU as defined in
    # [The alter_context PDU](https://pubs.opengroup.org/onlinepubs/9629399/chap12.htm#tagcjh_17_06_04_01)
    class AlterContext < BinData::Record
```